assets/css/pets-love.css

```diff
@@ -0,0 +1,38 @@
+.pets-love-wrapper{
+    display:flex;
+    height:90vh;
+}
+.pets-love-heading{
+    width:30%;
+    padding:30px;
+}
+.pets-love-heading h4{
+    margin-bottom:20px;
+    text-transform: uppercase;
+    letter-spacing: 4px;
+    padding-top:300px;
+}
+.pets-love-photo img{
+    padding:75px 0 0 0;
+}
+.pets-love-photo{
+    width:40%;
+}
+img{
+    width:100%;
+    height:75%;
```

Comment:
Po co ustawiać height? Tłumaczyłem Ci że jedyne co możesz zrobić w ten sposób to zepsuć proporcje. Dodatkowo wydaje mi się że ten height nie ma prawa działać w tym przypadku (moge się mylić - wtedy wytłumacz to zastosowanie)

Reply:
Poprawione